app/src/main/java/org/fossasia/susi/ai/MainApplication.java

```diff
@@ -23,6 +25,14 @@ public static MainApplication getInstance() {
     @Override
     public void onCreate() {
         super.onCreate();
+
```

Comment:
@amitiwary999 These 2 empty lines (1 in the starting and 1 at the end) are there to separate leak canary code from other code. This is to prevent accidental deletion of other useful code when we'll remove the leak canary code in future.